### ISSUE TITLE
Rocket League support broken after latest RL update

### ISSUE THREAD

**Issue by SPACEDgod** [SpoinkyNL/Artemis#163]
The boost visualizer was GREAT for Rocket League! So easy to get an idea of your boost through your peripheral vision. Unfortunately, the Rocket League Rumble update broke support, so now it only shows the flashing red/orange that it previously showed when boost was empty. I've updated to the latest version of Artemis but it is still not working.

Please fix this, I really love this app (I would like to add a genuine "Thank you!" for adding so much extra functionality to my Corsair Strafe RGB)! So far I've used it on RL, OW, and The Witcher 3, but RL is my game of choice.

**Comment by SpoinkyNL** [comment]
Hey, I've not updated this yet because I want to expand Rocket League support to include some more stuff (like team, score, how far your powerup is with charging up etc..) but this takes a bit more time which I currently don't seem to have :c

**Comment by SpoinkyNL** [comment]
1.4 includes fixed RL support :)

**Comment by SPACEDgod** [comment]
Amazing, thanks so much! Works flawlessly again, although one time I
launched Rocket League and it seemed to be running slowly, so I checked CPU
usage in task manager and it seemed like there was a background process of
Artemis using about 40% of my CPU (i5 4670k) so I ended the process but
Artemis kept working perfectly with Rocket League. Thanks again, this makes
having an RGB keyboard so much more user friendly!
